app/assets/stylesheets/recipe_finder.css.scss

```diff
@@ -57,12 +57,12 @@
   background: $color-airbnb-red;
   font-family: $font-airbnb;
   // box-shadow: 1px 1px 5px $color-airbnb-font;
-  border: 1px solid $color-airbnb-grey;
+  border: 1px solid $color-airbnb-red;
```

Comment:
Like @paulld said, I would just name `$color-airbnb-red` as `$main-theme-color` or something like that because if one day you decided to not follow Airbnb, then nothing makes sense.